src/OpenCensus.Exporter.Stackdriver/Implementation/StackdriverStatsExporter.cs

```diff
@@ -45,9 +47,8 @@ internal class StackdriverStatsExporter
         private const string CUSTOM_METRIC_DOMAIN = "custom.googleapis.com/";
         private const string CUSTOM_OPENCENSUS_DOMAIN = CUSTOM_METRIC_DOMAIN + "opencensus/";
 
-        // TODO - zeltser - figure out how to extract OpenCensus package version
-        private const string USER_AGENT = "opencensus-csharp [0.0]";
         private const string USER_AGENT_KEY = "user-agent";
+        private static string USER_AGENT;
```

Comment:
I wonder why Stylecop doesn't complain on these names. They are violating C# coding guidelines. Perhaps we can look at this at some point. Just a general comment, not action required

Reply:
I don't use stylecop (I can install it), so it didn't flag it for me. I deliberately named it as a constant so people don't use it in a wrong way.